stable/memcached/Chart.yaml

```diff
@@ -1,11 +1,13 @@
 name: memcached
-version: 2.2.0
-appVersion: 1.5.6
+version: 2.3.2
```

Comment:
Why did the chart version go up to `2.3.2` instead of `2.3.0`?

Reply:
Thanks @mattfarina, we use this chart in production, and while it has been waiting for review it has gone through several rounds of internal improvements, so the version has been bumped several times in our repository. The original PR only bumped it 2.2.0 to 2.2.1 to add some settings, but we've done a lot of additional work on this that we wanted to try to contribute back.

I assume if anything gets merged we'll update this in the PR based on whatever the current master version is.